pkg/apis/apps/v1beta1/zz_generated.conversion.go

```diff
@@ -291,7 +290,8 @@ func autoConvert_v1beta1_DeploymentSpec_To_extensions_DeploymentSpec(in *v1beta1
 		return err
 	}
 	out.Selector = (*meta_v1.LabelSelector)(unsafe.Pointer(in.Selector))
-	if err := api_v1.Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
I run `make generated_files`, nothing changed. I also run `hack/update-generated-docs.sh`, `hack/update-codegen.sh`, `hack/update-codecgen.sh`, etc (not sure if they're directly related), also nothing changed.